### PR TITLE
Unrelated to "Religion".

### PR DESCRIPTION
**Pull request by Julian-O**
Seems to be set to "Religion" as a joke, but this is polluting the PyPi name spaces. Cleaning up.

Note: Untested,

### CODE REVIEW

**Comment by tusbar** [comment]
getmillipede is a religion though, @evadot?

**Comment by evadot** [comment]
From https://pypi.python.org/pypi?%3Aaction=list_classifiers

Both are allowed so why do you want to remove them ?
(And yes getmillipede is a religion).

**Comment by Julian-O** [comment]
I am politely asking you to remove it because when you go to see which Python packages are on the topic of Religion, getmillipede turns up, polluting the name space.

Your claims that getmillipede is a religion are, of course, facetious. You might equally argue it belongs to about fifty other categories as well. Pastafarians have a real purpose in undermining the role of Creationism in the science class, but Millipedists are hardly striking a satirical blow against religion in the religion category of PyPi.

**Comment by edouardb** [comment]
From the really serious article  `10 Fascinating Facts About Millipedes`  on thoughtco.com https://www.thoughtco.com/fascinating-facts-about-millipedes-1968229

10. MILLIPEDES WERE THE FIRST ANIMALS TO LIVE ON LAND.

So yes, getmillipede is a god, a religion and also the father of every religions.

**Review by louiz — CHANGES_REQUESTED** [review]
I don’t have any good reason, but I do not approve of this change. Keep it a religion.

File: setup.py [hunk 1]
```diff
@@ -43,9 +43,8 @@ def get_version():
     classifiers=[
         'Development Status :: 5 - Production/Stable',
 
-        'Intended Audience :: Religion',
-        'Topic :: Religion',
-
+        'Intended Audience :: Other Audience',
```
Comment (tusbar) [inline setup.py:46]: This change introduces backward compatibility issues with the previous versions of the package.

Most of our users are using it in a strictly religious way. I do not believe that removing that from them would be a good choice.

**Review by tusbar — CHANGES_REQUESTED** [review]

**Comment by brmzkw** [comment]
Because you asked me to review this PR, my opinion: I really don't understand why you are being assholes on something that seems to affect someone. Just remove the classifier, no one cares.

**Comment by Julian-O** [comment]
- No, there are no backward compatibility issues.

- No, the category is not for tools that are used religiously.

- No, biological facts about millipedes are hardly relevant - especially for a tool that prints **centipedes** and calls them millipedes.

At this stage, I feel like I am dealing with people with a sense of humour about the level of high school students, so I will bow out of further trolling.

**Comment by evadot** [comment]
This merge request makes me sorry for two things:

1 - The fact that us, millipedians, aren't take seriously among the python community (which in fact don't surprise me a lot since python will surely eat millipede in the wild)
2 - The fact that one of our funding member as rejected us to join the Lady Bug church (which is nothing more that a sect telling lie to millipedians to steal their money), that's why the board members of the Millipede Church have chosen to remove @brmzkw from the github organization.

So to be clear about the merge request, no we won't merge it but thanks for putting up such change as it allowed us to see the light on one of our member, and maybe (but I honestly prefer it won't) will too on other ones.

I'll for now leave the MR open for giving a chance to the members that have joined the Lady Bug church to step up.

Thanks.

**Pull request closed without merging**